app/services/hyrax/thumbnail_path_service.rb

```diff
@@ -5,7 +5,7 @@ class << self
       # @param [#id] object - to get the thumbnail for
       # @return [String] a path to the thumbnail
       def call(object)
-        return default_image unless object.try(:thumbnail_id)
+        return default_image unless object.try(:thumbnail_id)&.present?
```

Comment:
```suggestion
        return default_image unless object.try(:thumbnail_id).present?
```

Reply:
`nil` responds to `#present?`. i doubt this matters beyond visual clutter, but let's skip the `&`